tests/python/relay/test_op_level5.py

```diff
@@ -270,8 +270,8 @@ def verify_get_valid_counts(dshape, score_threshold, id_index, score_index):
             intrp = relay.create_executor("debug", ctx=ctx, target=target)
             out = intrp.evaluate(func)(np_data)
             tvm.testing.assert_allclose(out[0].asnumpy(), np_out1, rtol=1e-3, atol=1e-04)
-            # get_valid_count for cuda doesn't do data rearrangement
-            if target == 'cuda':
+            # get_valid_count for cuda, opencl doesn't do data rearrangement
+            if target in ['cuda', 'opencl']:
                 return
```

Comment:
Returning here looks wrong to me.  The test in the below link doesn't work for OpenCL too because we don't do data rearrangement for GPU nms implementation.
https://discuss.tvm.ai/t/nms-compile-fails-for-cuda-target-but-works-fine-for-llvm-target/7045/2

Probably, we should fix non_max_suppression for GPU first?

Reply:
OpenCL uses the same implementation as CUDA. The CUDA implementation of `get_valid_counts` was changed to no longer rearrange the output of `get_valid_counts` because it will be rearranged by NMS later anyway. This gives the correct output for NMS.

That issue with NMS looks to be a separate issue.

Reply:
See https://github.com/apache/incubator-tvm/pull/5339

Reply:
Thanks for your explanation.  Actually, I've successfully build NMS if I revert the change in #4312.